dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -3228,6 +3236,66 @@ internal class RumViewScopeTest {
         assertThat(result).isSameAs(testedScope)
     }
 
+    @Test
+    fun `𝕄 send View update 𝕎 onVitalUpdate()+handleEvent(KeepAlive) {frameRate}`(
+        forge: Forge
+    ) {
+        // Given
+        val frameRates = forge.aList { aDouble(0.0, 60.0) }.sorted()
+        val listenerCaptor = argumentCaptor<VitalListener> {
+            verify(mockFrameRateVitalMonitor).register(capture())
+        }
+        val listener = listenerCaptor.firstValue
+
+        // When
+        var sum = 0.0
+        var min = 60.0
+        var max = 0.0
+        var count = 0
+        frameRates.forEachIndexed { _, value ->
```

Comment:
very minor: probably can be just `forEach`, or `index` was supposed to be used somehow?